scipy/sparse/linalg/_isolve/iterative.py

```diff
@@ -300,16 +301,17 @@ def bicgstab(A, b, x0=None, tol=1e-5, maxiter=None, M=None, callback=None, atol=
                'Alternatively, ``A`` can be a linear operator which can\n'
                'produce ``Ax`` using, e.g.,\n'
                '``scipy.sparse.linalg.LinearOperator``.',
-               footer="""
-
+               footer="""\
                Examples
                --------
                >>> from scipy.sparse import csc_matrix
                >>> from scipy.sparse.linalg import cg
-               >>> R = np.random.randn(5, 5)
-               >>> positive_definite = R @ R.T
-               >>> A = csc_matrix(positive_definite)
-               >>> b = np.random.randn(5)
+               >>> P = np.array([[4, 0, 1, 0],
+               ...               [0, 5, 0, 0],
+               ...               [1, 0, 3, 2],
+               ...               [0, 0, 2, 4]]])
```

Comment:
We got an extra parenthesis.
```suggestion
               ...               [0, 0, 2, 4]])
```